pandas/core/generic.py

```diff
@@ -10818,9 +10818,12 @@ def describe(
         ----------
         percentiles : list-like of numbers, optional
             The percentiles to include in the output. All should
-            fall between 0 and 1. The default is
-            ``[.25, .5, .75]``, which returns the 25th, 50th, and
-            75th percentiles.
+            fall between 0 and 1. Here are the options:
+
+            - A list-like of numbers : To include the percentiles listed. If
+              that list is empty, no percentiles will be returned.
+            - None (default) : To include the default percentiles, which are the
+              25th, 50th, and 75th ones.
```

Comment:
I think you can just keep the old description but clarify `The default, ``None``, will automatically return the 25th, 50th, and 75th percentiles.`